deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkProcessor.java

```diff
@@ -307,12 +322,28 @@ public int compare(DeepLinkAnnotatedElement element1, DeepLinkAnnotatedElement e
         .addField(registry)
         .addMethod(parseMethod)
         .build();
-
     JavaFile.builder(packageName, deepLinkLoader)
         .build()
         .writeTo(filer);
   }
 
+  private String buildPrefixString(String[] prefixes) {
+    StringBuilder prefixArgumentBuilder = new StringBuilder("new String[]{");
+    for (int j = 0; j < prefixes.length; j++) {
+      String prefix = prefixes[j];
+      prefixArgumentBuilder
+              .append("\"")
+              .append(prefix)
+              .append("\"");
+
+      if (j < prefixes.length - 1) {
+        prefixArgumentBuilder.append(", ");
+      }
+    }
+    prefixArgumentBuilder.append("}");
```

Comment:
Another reason to use a `List` so you don't have to do this manual `StringBuilder` stitching